projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -64,7 +64,6 @@ public static function get_all_product_data() {
 
 		// If site has backups and the realtime backup capability, add latest undo event
 		if (
-			$body->backups->last_finished_backup_time &&
 			in_array( 'backup-realtime', $capabilities->data['capabilities'], true )
 		) {
 			$body->backups->last_undoable_event = self::get_site_backup_undo_event();
```

Comment:
Another thought here, if we know the site has a backup capability, do we need to call the first API endpoint to `jetpack-prodcut-data`? Could we just look up the last rewindable event?

Reply:
I had thought about this, but we are still set up here for the "products-data" endpoint to be the main API call for all the cards, not just backup. I know we want to move away from that, but I didn't know how much refactoring we wanted to do right now 🤔 

I think if we were to move that conditional and not call the API if it is true, we'd want to go with a larger refactor right now because the way things are set up right now, this would prevent any other product data from being queried. I know we aren't pulling any other product data right now but it could be confusing for the next person if we leave things as they are since it looks like that is what it is for. I hope I am explaining my thoughts clearly enough

Do you think this should be a larger refactor to move away from the product data API endpoint right now?

Reply:
To add some notes from our conversation:

I think we can leave this as-is in the current diff, but we will revisit what would be involved in refactoring this and what would make sense to tackle before the next release